dice-game-ui-angular/src/app/start-game/start-game.component.ts

```diff
@@ -1,17 +1,32 @@
 import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { SimpleGameEngine } from '../../../../engine/simple-game-engine';
+import { GameStorageService } from '../services/game-storage-service';
+import { LocalGameStorageService } from '../services/local-game-storage-service';
 
 @Component({
   selector: 'app-start-game',
   templateUrl: './start-game.component.html',
   styleUrls: ['./start-game.component.scss'],
+  providers: [{ provide: GameStorageService, useClass: LocalGameStorageService }],
```

Comment:
Also is not needed as it will be declared in a module.

Reply:
Removed